app/models/orders.py

```diff
@@ -32,21 +32,20 @@ class Orders(db.Model):
 
     def __init__(
             self,
-            id,
+            _id,
             date_submitted,
             date_received,
-            order_types,
+            _order_types,
             multiple_items,
             _next_suborder_number=1,
             confirmation_message=None,
-            client_data=None,
-    ):
-        self.id = id
+            client_data=None):
+        self.id = _id
```

Comment:
Shouldn't we rename the self `id` attribute to something that doesn't shadow the builtin?

Reply:
I think this should be fine for now. We can tackle on refactoring attributes to not use builtin variables on the next go around?